src/dependencies.ts

```diff
@@ -51,16 +54,20 @@ export class RuntimeDependencies implements Dependencies {
         }
     }
 
-    async execCommand(command: string, envVars: EnvironmentVariables = {}): Promise<number> {
+    async execCommand(command: string, envVars: EnvironmentVariables = {}): Promise<CommandOutput> {
         try {
-            return await exec.exec(command, [], {
+            return await exec.getExecOutput(command, [], {
                 env: mergeWithProcessEnvVars(envVars),
                 ignoreReturnCode: true,
                 failOnStdErr: false
             })
         } catch (err) {
             // A try/catch is needed here due to issue: https://github.com/actions/toolkit/issues/1625
-            return 127
+            return {
+                exitCode: 127,
```

Comment:
Should we declare this as a constant somewhere else, instead of just having the hardcoded number in the code?

Reply:
I don't think so. This is the only place it is used and it is a well known linux standard where 127 means command not found. Hopefully this catch statement will be temporary anyway if they fix this issue.

Reply:
Agree with @jfeingold35 . 127 feels like a magic number, especially for someone who hasn't worked with linux.

Reply:
I can update the 127 to make it a constant.

And I have tested this on all github hosted runners - all platforms.

Reply:
Done.